packages/portal/frontend/html/style.css

```diff
@@ -107,4 +107,13 @@ stdio viewer Page
     right: 0;
     bottom: 0;
     left: 0;
-}
\ No newline at end of file
+}
+
+#adminClasslistConfirmHeader .dialog {
+    width: 80%;
+}
+
+#adminClasslistConfirmHeader ons-list {
+    overflow-y: auto;
+    max-height: 450px;
```

Comment:
Do we want to do this with `px` or with `%`? I ask because `px` will be fine for computers but might be more problematic on mobile (and we're seeing a lot of mobile access from both the TAs, and from, well, me :) )

Reply:
Sounds great @rtholmes .

We can make mobile and tablet support a convention going forward. 

Using percentage causes a lot of issues via inheritance from Onsen UI components, so we are using viewport height percentage instead.